packages/amplify-graphql-transformer-core/src/transform-host.ts

```diff
@@ -0,0 +1,301 @@
+import {TransformHostProvider} from '@aws-amplify/graphql-transformer-interfaces';
```

Comment:
```suggestion
import { TransformHostProvider } from '@aws-amplify/graphql-transformer-interfaces';
```

There are a few more of these below.